requirements-dev.in

```diff
@@ -23,6 +23,7 @@ Faker==17.5.0
 fakeredis[lua]==2.23.3
 freezegun==1.2.2
 inline-snapshot==0.12.*
+multidict==6.0.5
```

Comment:
Since we don't import `multidict` anywhere, let's explicitly comment on this requirement's line why it's needed

Reply:
That makes sense, just added the comment